proteus/mprans/MCorr.py

```diff
@@ -102,6 +105,12 @@ def attachModels(self,modelList):
             self.ebq_H_vof = modelList[self.VOFModelIndex].ebq[('u',0)]
         else:
             self.ebq_H_vof = None
+
+        #: YY
+        self.q_H_phi_old = numpy.zeros_like(modelList[self.VOFModelIndex].q[('u',0)])
```

Comment:
we want `self.q_phi_old = numpy.zeros_like(modelList[self.levelSetModelIndex].q[('u',0)])`